sdk/communication/Azure.Communication.Common/src/MicrosoftTeamsUserIdentifier.cs

```diff
@@ -8,28 +8,38 @@ namespace Azure.Communication
     /// <summary>Represents a Microsoft Teams user.</summary>
     public class MicrosoftTeamsUserIdentifier : CommunicationIdentifier
     {
+        /// <summary>The id of the Microsoft Teams User identifier.</summary>
+        public string? Id { get; }
+
         /// <summary>The id of the Microsoft Teams user. If the user isn't anonymous, the id is the AAD object id of the user.</summary>
         public string UserId { get; }
 
         /// <summary>True if the user is anonymous, for example when joining a meeting with a share link.</summary>
         public bool IsAnonymous { get; }
 
+        /// <summary> The cloud that the identifier belongs to. </summary>
```

Comment:
nit: extra space

```suggestion
        /// <summary>The cloud that the identifier belongs to. </summary>
```

Reply:
That is a autogen issue. But does not affect the generated doc. It is ok.